app/scripts/lib/channel-server-client.js

```diff
@@ -0,0 +1,177 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import a256gcm from './crypto/a256gcm';
+import { pick } from 'underscore';
+import base64 from './base64';
+import Cocktail from 'cocktail';
+import { DEVICE_PAIRING_CHANNEL_KEY_BYTES } from './constants';
+import importFxaCryptoDeriver from './crypto/deriver';
+import { Model } from 'backbone';
+import UserAgentMixin from './user-agent-mixin';
+import hkdf from './crypto/hkdf';
+
+const CHANNEL_KEY_INFO = 'identity.mozilla.com/picl/v1/pair/encryption-key';
+
+const CONFIRMATION_CODE_INFO = 'identity.mozilla.com/picl/v1/pair/confirmation-code';
+const CONFIRMATION_CODE_LENGTH = 4;
+
+export default class ChannelServerClient extends Model {
+  constructor (attrs = {}, options = {}) {
+    if (attrs.channelId) {
+      attrs.type = 'supp';
+    } else {
+      attrs.type = 'auth';
```

Comment:
Remove the auth portions of the client, it's no longer needed since Desktop does all that work.